File: Lista2/Java/Pieciokat.java
public class Pieciokat extends Figura
{
    private double bok;

    Pieciokat(double bok)
    {
        this.bok = bok;
    }

    public double obwod()
    {
        return (5 * bok);
    }

    public double pole()
    {
        return (Math.sqrt(25 + (10 * Math.sqrt(5))) * Math.pow(bok, 2)/4);
    }
}

File: Lista2/C++/Pieciokat.cpp
#include <iostream>
#include <cmath>
#include "Pieciokat.h"

Pieciokat::Pieciokat(double bok)
{
    this->bok = bok;
}

double Pieciokat::obwod()
{
    return (5 * bok);
}

double Pieciokat::pole()
{
    return (sqrt(25 + 10 * sqrt(5))  * pow(bok, 2) / 4);
}

Pieciokat::~Pieciokat()
{
	
}